extra/language/rust.nix

```diff
@@ -5,33 +5,40 @@ in
 with lib;
 {
   options.language.rust = {
-    packageSet = mkOption {
-      # FIXME: how to make the selection possible in TOML?
-      type = types.attrs;
-      default = pkgs.rustPackages;
-      defaultText = "pkgs.rustPlatform";
-      description = "Which rust package set to use";
-    };
```

Comment:
As a user, I would expect *some* version of rust to be installed with the module.

Reply:
Completely understand your concern here. Not sure what the best solution is though.
The way I've seen it in most places, is that people use an additional overlay that brings the rust toolchain they want with an additional package. Since it's not compatible with this, I removed it. Happy to do it another way though.

Reply:
As a user, I would expect to be able to pick the rust version here. Use the version from nixpkgs (there are valid use-cases for this), or fenix, ... 

The expectation is that whatever the source, the packageSet interface would be the same. That's something to fix with the other projects so they are all aligned.

On top of that, I think an acceptable escape hatch would be to allow `null` as a value so the user can disable that part.